test/verify/check-system-info

```diff
@@ -931,6 +915,37 @@ password=foobar
         b.wait_in_text(".pf-c-menu__item.pf-m-selected", "Custom crypto policy")
         b.click("#crypto-policy-dialog button.pf-c-button.pf-m-link")
 
+    @skipImage("crypto-policies not avaiable", "fedora-coreos", "debian-stable", "debian-testing", "ubuntu-2204", "ubuntu-2004", "ubuntu-stable", "arch")
```

Comment:
avai*l*able; it's also wrong in the existing skipImage already, can you please fix while you are at it?